etc/config.sample.toml

```diff
@@ -1,125 +1,161 @@
-
-# Broker configuration. Brokers are nodes which participate in distributed
-# consensus.
-[broker]
-enabled = true
-# Where the Raft logs are stored. The user running InfluxDB will need read/write access.
-dir  = "/var/opt/influxdb/raft"
-truncation-interval = "10m"
-max-topic-size = 52428800
-max-segment-size = 10485760
-
-# Raft configuration. Controls the distributed consensus system.
-[raft]
-apply-interval = "10ms"
-election-timeout = "5s"
-heartbeat-interval = "100ms"
-reconnect-timeout = "10ms"
-
-# Data node configuration. Data nodes are where the time-series data, in the form of
-# shards, is stored.
 [data]
-enabled = true
-dir = "/var/opt/influxdb/db"
+  dir = "/var/opt/influxdb/data"
+
+###
+### [cluster]
+###
+### Controls non-Raft cluster behavior, which generally includes how data is
+### shared across shards.
+###
+
+[cluster]
+  shard-writer-timeout = "5s"
+
+###
+### [retention]
+###
+### Controls the enforcement of retention policies for evicting old data.
+###
+
+[retention]
+  enabled = true
+  check-interval = "10m"
+
+###
+### [admin]
+###
+### Controls the availability of the built-in, web-based admin interface.
+###
+
+[admin]
+  enabled = true
+  bind-address = ":8083"
+
+###
+### [http]
+###
+### Controls how the HTTP endpoints are configured. These are the primary
+### mechanism for getting data into and out of InfluxDB.
+###
+
+[http]
+  enabled = true
+  bind-address = ":8086"
+  auth-enabled = false
+  log-enabled = true
+  write-tracing = false
+  pprof-enabled = false
+
+###
+### [[graphite]]
+###
+### Controls one or many listeners for Graphite data.
+###
+
+[[graphite]]
+  enabled = false
+  # protocol = ""
+  # bind-address = ""
+
+###
+### [collectd]
+###
+### Controls the listener for collectd data.
+###
 
-# Auto-create a retention policy when a database is created. Defaults to true.
-retention-auto-create = true
+[collectd]
+  enabled = false
+  # bind-address = ""
+  # database = ""
+  # typesdb = ""
 
-# Control whether retention policies are enforced and how long the system waits between
-# enforcing those policies.
-retention-check-enabled = true
-retention-check-period = "10m"
+###
+### [opentsdb]
+###
+### Controls the listener for OpenTSDB data.
+###
 
-# Configuration for snapshot endpoint.
-[snapshot]
-enabled = true # Enabled by default if not set.
+[opentsdb]
+  enabled = false
+  # bind-address = ""
+  # database = ""
+  # retention-policy = ""
 
-[logging]
-write-tracing = false # If true, enables detailed logging of the write system.
-raft-tracing = false # If true, enables detailed logging of Raft consensus.
-http-access = true # If true, logs each HTTP access to the system.
+###
+### [udp]
+###
+### Controls the listener for InfluxDB line protocol data via UDP.
+###
+
+[udp]
+  enabled = false
+  # bind-address = ""
+  # database = ""
+  # batch-size = 0
+  # batch-timeout = "0"
+
+###
+### [monitoring]
+###
+### Controls the anonymous statistics reporting function of InfluxDB. At start
```

Comment:
I don't see any actual reference to this flag in the configs. I say this because people can be sensitive about this, and we should be crystal clear on how to turn it off.